src/BudgetManager/BudgetManager.SDK/DTOs/CategoryDTOs/UpdateCategoryDTO.cs

```diff
@@ -1,15 +0,0 @@
-﻿using BudgetManager.Model;
-using BudgetManager.Model.Enums;
-
-namespace BudgetManager.SDK.DTOs.CategoryDTOs
-{
-    public class UpdateCategoryDTO
-    {
-        public Guid CategoryId { get; set; }
-        public string Name { get; set; }
-        public decimal Limit { get; set; }
-        public LimitPeriods? LimitPeriod { get; set; }
-        public OperationType CategoryType { get; set; }
-        public string Color { get; set; }
-    }
-}
```

Comment:
We need this DTO, but Operation Type should be removed, because we can not change it
Also in this case we need to have separeted DTO for Adding Category (DTO for this operations should be named simply and understandable )